bundles/org.openhab.core.io.transport.modbus/src/main/java/org/openhab/core/io/transport/modbus/ModbusReadRequestBlueprint.java

```diff
@@ -126,7 +119,7 @@ public boolean equals(@Nullable Object obj) {
             return false;
         }
         ModbusReadRequestBlueprint rhs = (ModbusReadRequestBlueprint) obj;
-        return new EqualsBuilder().append(slaveId, rhs.slaveId).append(functionCode, rhs.functionCode)
-                .append(start, rhs.start).append(length, rhs.length).isEquals();
+        return functionCode == rhs.functionCode && length == rhs.length && maxTries == rhs.maxTries
```

Comment:
Original implementation did not compare maxTries. I think it is simply a mistake, cannot see any harm with adding it to equals

Reply:
Ok. Will fo

Reply:
I reverted this just in case.